### PR TITLE
Feature/us3367 pick a group connector

### PR DESCRIPTION
**Pull request by silbermm**

### CODE REVIEW

File: CI/SQL/20160318151401_US3367-update-project-page-view.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+USE [MinistryPlatform]
```
Comment (andycanterbury) [inline CI/SQL/20160318151401_US3367-update-project-page-view.sql]: This file and 20160323151501_US3367-update-group-connectors-by-organization-page-view.sql seem to both be editing the field list for the same page view. Combine into one so we don't overwrite each other?

File: Gateway/Crossroads.AsyncJobs/default.aspx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿<!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml">
+<html xmlns="http://www.w3.orgId/1999/xhtml">
```
Comment (andycanterbury) [inline Gateway/Crossroads.AsyncJobs/default.aspx]: why did this change? Looks like a find/replace all error.

File: Gateway/CrossroadsStripeOnboarding/Migrations/201512071726262_Initial.resx [hunk 3]
```diff
@@ -59,8 +59,8 @@
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
-    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" />
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.orgId/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```
Comment (andycanterbury) [inline Gateway/CrossroadsStripeOnboarding/Migrations/201512071726262_Initial.resx]: find and replace error?

File: Gateway/CrossroadsStripeOnboarding/Migrations/201512071726262_Initial.resx [hunk 4]
```diff
@@ -59,8 +59,8 @@
...
+    <xsd:import namespace="http://www.w3.orgId/XML/1998/namespace" />
```
Comment (andycanterbury) [inline Gateway/CrossroadsStripeOnboarding/Migrations/201512071726262_Initial.resx]: find and replace error?

File: Gateway/MinistryPlatform.Translation/Services/GroupConnectorService.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+        public List<MpGroupConnector> GetGroupConnectorsForOrganization(int organizationId, int initiativeId)
+        {
+            var searchString = string.Format(",,,,{0},,{1}", organizationId, initiativeId);
+            return GetGroupConnectors(searchString);
+        }
+
+        private List<MpGroupConnector> GetGroupConnectors(string searchString)
+        {
+            var token = ApiLogin();
```
Comment (andycanterbury) [inline Gateway/MinistryPlatform.Translation/Services/GroupConnectorService.cs]: Pass in a token so translation is secure and gateway can choose to use apilogin if needed.

File: crossroads.net/app/common/profile/picture/profilePicture.html [hunk 6]
```diff
@@ -1,8 +1,14 @@
...
-</div>
+<div ng-class="picture.wrapperClass">
+  <label for="fileInputModal" class="profile-pic" ng-click="picture.openModal()" ng-hide="picture.disallowChange" >
+      <img class="profile-image" ng-class="picture.imageClass"
+        ng-src="{{picture.path}}" alt="Profile Photo"
+        err-src="{{picture.defaultImage}}"/>
+      <span class="change-photo">{{picture.buttonText}}</span>
+  </label>
+
+ <img class="profile-image" ng-class="picture.imageClass"
```
Comment (andycanterbury) [inline crossroads.net/app/common/profile/picture/profilePicture.html]: Is this redundant with the img inside the label tag?

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js [hunk 7]
```diff
@@ -1,240 +1,249 @@
...
+        parent: 'goCincinnati',
+        abstract: true
+      })
+      .state('go-volunteer.city', {
+        parent: 'goCincinnati',
+        url: '/go-volunteer/:city',
+        template: '<go-volunteer-city></go-volunteer-city>',
+        data: {
+          meta: {
+            title: 'Some Title',
```
Comment (andycanterbury) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: Does this need to be changed to not be 'Some Title'?

File: crossroads.net/app/common/profile/picture/profilePicture.html (hunk 6 above)
Comment (silbermm) [inline crossroads.net/app/common/profile/picture/profilePicture.html]: No, it looks redundant, but one image is wrapped in a label and that should be displayed under one condition. The image without the label should be shown when another condition happens

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js (hunk 7 above)
Comment (silbermm) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: There is a story for this

File: Gateway/MinistryPlatform.Translation/Services/GroupConnectorService.cs [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+        public List<MpGroupConnector> GetGroupConnectorsForOrganization(int organizationId, int initiativeId, string token)
+        {
+            var searchString = string.Format(",,,,{0},,{1}", organizationId, initiativeId);
+            return GetGroupConnectors(searchString, token);
+        }
+
+        private List<MpGroupConnector> GetGroupConnectors(string searchString, string token)
+        {
+            //var token = ApiLogin();
```
Comment (andycanterbury) [inline Gateway/MinistryPlatform.Translation/Services/GroupConnectorService.cs]: Commented code.  :-P

**Pull request merged**